src/AzureIoTHub.Portal.Client/Pages/EdgeDevices/CreateEdgeDevicePage.razor

```diff
@@ -304,10 +304,21 @@
         {
             this._edgeModel = edgeModel;
 
+            this.EdgeDevice = new IoTEdgeDevice()
+                {
+                    DeviceId = this.EdgeDevice.DeviceId,
+                    DeviceName = this.EdgeDevice.DeviceName,
+                    ModelId = edgeModel?.ModelId,
+                    ImageUrl = edgeModel?.ImageUrl,
```

Comment:
Please add also a unit test to verify that the model image is displayed on the page